examples/coordinates/plot_galactocentric-frame.py

```diff
@@ -108,9 +109,10 @@
 # We still have to assume a line-of-sight velocity for the Galactic center,
 # which we will again take to be 11 km/s:
 vx = 11.1 * u.km/u.s
+v_sun2 = u.Quantity([vx, vy, vz])
```

Comment:
The need to cast Quantity here is not obvious.
```suggestion
v_sun2 = u.Quantity([vx, vy, vz])  # List of Quantity -> a single Quantity
```